src/10_functions.py

```diff
@@ -1,12 +1,17 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even():
 
 # Read a number from the keyboard
-num = input("Enter a number: ")
-num = int(num)
+    num = input("Enter a number: ")
+    num = int(num)
 
 # Print out "Even!" if the number is even. Otherwise print "Odd"
-
+    if(num % 2 == 0):
```

Comment:
FYI, you don't need to wrap this statement after the if in parentheses in Python, which is a big part of why I always forget them in JS. Nice looking function overall